### ISSUE TITLE
Play with the CNN Structure to get it working well

### ISSUE THREAD

**Issue by alexwdong** [alexwdong/IncubatorCVProject#13]
This task would involve tweaking the Basic CNN structure we have already set up, (by changing convolution parameters, adding more layers, using dropout, batch normalization, etc), to get a performance improvement. 

This would be done on the Stanford Dogs Dataset

For baseline,  after training 20 epochs with my current set up, i'm getting a validation accuracy of ~.03
